shared/chat/inbox/container.tsx

```diff
@@ -187,7 +206,13 @@ const Connected = Container.namedConnect(
         bigRows.push(teamBuilder)
       }
     }
-    const rows: Array<Types.ChatInboxRowItem> = [...smallRows, ...divider, ...bigRows]
+    const nextRows: Array<Types.ChatInboxRowItem> = [...smallRows, ...divider, ...bigRows]
+    let rows = nextRows
+    // TODO better fix later
+    if (isEqual(rows, cachedRows)) {
```

Comment:
how rows are built and distributed to sub components could be made better but that's too big a diff for now so instead of just keep a cache in case of deep equality. better to pay for this here vs all the downstream thrash